docs/guides/create_service_existing_agent.md

```diff
@@ -1,3 +1,5 @@
+{{check_untagged_doc()}}
```

Comment:
What's this about? I don't get it

Reply:
The idea is to get a Warning on top of the page whenever the docs are compiled outside a tagged version. The 'macros' package allows to execute code inside the docs on compile-time.